src/main/java/com/twilio/http/Request.java

```diff
@@ -285,7 +285,12 @@ private static String encodeParameters(final Map<String, List<String>> params) {
                         continue;
                     }
 
-                    String encodedValue = URLEncoder.encode(value, "UTF-8");
+                    String encodedValue;
+                    if (value.startsWith("+")) {
```

Comment:
This is not the proper solution and there are many other characters which this would not solve for. Proper changes are needed in `com.twilio.http.Request.buildURL()`

Reply:
`buildURL()` appears to be constructing the URL properly (I compared the current method with the method prior to #558). The issue seems to be within `encodeParameters()` where each parameter is encoded. In the case of a phone number, it appears we should not encode the + sign. `contructURL()` is where we add the parameters to the final URL.

Reply:
The first page works fine which is where `encodeParameters` is called. The second page fails because `buildURL` is re-encoding the already-encoded parameters.

Reply:
I'm not seeing where `buildURL` is re-encoding the parameters. Might be easier to talk through it?

I added code to decode each value first before encoding. That seems to work locally, checking tests.